setup/ubuntu/bootstrap.sh

```diff
@@ -40,6 +40,7 @@ apt-get install -y libsasl2-dev
 #Saml dependency
 apt-get install -y xmlsec1
 
+pip install --upgrade pip
```

Comment:
Last I tried it, on Ubuntu 14.04 (which we currently use for Docker and AWS images) this doesn't work, because it fails to upgrade a system package installed pip...

Maybe we can wrap this with Ubuntu version check?

Eventually we will migrate all our images to 16.04, but until then...